styles.css

```diff
@@ -46,31 +59,49 @@ nav {
 }
```

Comment:
- [ ] As per project requirement you are to use the same background image for your project. Kindly replace your background image with the design image